.github/workflows/ci.yml

```diff
@@ -83,13 +83,6 @@ jobs:
           dotnet-version: 3.1.407
       - name: Pin .NET Core SDK
         run: dotnet new globaljson --sdk-version 3.1.407
-      - name: Restore tools
-        run: dotnet tool restore
-        working-directory: ./Src
```

Comment:
Are you sure that these lines here are not responsible for the current problems we are having on the Windows build machines?

Reply:
I only ask because the Windows CI build is still not able to find dotnet-xunit related things.... only on Windows.

Reply:
I have to say, I'm starting to think it might be related 😓
I'll investigate in my repo.

Reply:
I poked around a bit more and managed to reproduce this issue with the current master branch too. It seems to be quite random, but always happening on the Windows runners, and only when the Visual Studio offline NuGet source is enabled (which is the default). Disabling this NuGet source seems to solve the issue permanently, and I would recommend we do that, at least until GitHub fixes the Windows runner (which seems to have other issues in terms of NuGet restore, as it usually takes about 3 times longer than on Linux).

I've added a commit doing just that in this branch, but I'm happy to create a separate PR for it if you'd like.